frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -157,7 +157,7 @@ const DebugInformationComponent: React.FC<DebugInformationProps> = (props: Debug
     setCopyStatus(CopyStatus.NOT_COPIED);
   };
 
-  const parseConfig = (key: string, value: string): string | null => {
+  const parseConfig = (key: string, value: unkown): unkown | null => {
```

Comment:
```suggestion
  const parseConfig = (key: string, value: unknown): unknown | null => {
```
There was a typo in my first suggestion